src/server/server.ts

```diff
@@ -101,15 +104,32 @@ export async function startApplication(args: CLIArguments) {
                     .addListener(
                         'closed',
                         async () => {
-                            process.exit(0);
+                            process.exit();
                         }
                     )
                     .catch(console.error);
+
+                // Fix for Windows 10 not killing the demo apps on SIGINT
+                const signals: ('exit' | NodeJS.Signals)[] = ['exit', 'SIGINT', 'SIGTERM'];
+                signals.forEach((sig) => {
+                    // @ts-ignore type error on exit
+                    process.on(sig, () => cleanup([demo]));
```

Comment:
Don't most of the demo apps create separate windows/applications? I think this wouldn't actually (fully) close the demo in most cases?.. Is that a concern?